docs/Reference/Certificates/remoteProvisioning.md

```diff
@@ -0,0 +1,90 @@
+--8<-- "References/abbreviations.md"
+	
+## What is remote provisioning?
+
+Provisioning is the overall process of bringing a remote device under the management of the development system. The process involves activation and configuration of a device without physically touching it. 
+
+!!! Note Whatever 
+    Before provisioning, identify the environment in which the device will be managed. The environments include Public Cloud, Private Cloud, and Hybrid Cloud.
+
+The table below describes the process: 
+
+| Step       | Significance to Provisioning|  
+| :------------------------- | :--------------------- | 
+| 1. Creating a Client Initiated Remote Access (CIRA) configuration | This saved configuration will be used to configure the CIRA protocol for public cloud environments. |
+| 2. Setting up the AMT Profile and choosing an activation type | Choose from two modes, Admin Control Mode (ACM) and Client Control Mode (CCM). If ACM is chosen, add a provisioning certificate in Step 3. CCM does not require a certificate. |
+| 3. Adding a domain profile and associated provisioning certificate (signed or self-signed) | Upload a provisioning certificate. |
+| 4. Using the rpc-go client application to activate the remote device | Deploy and run the client as Administrator on the desired remote platform. Leverage the profile you created in Step 2.  |
+
+
+Figure 1 illustrates the steps below. Also, see Get Started
+
+To manage a device remotely, you must *provision* it, securely configure it to communicate with the development system, the provisioning server. The toolkit accomplishes secure provisioning with a process of authentication and a tunnel using Transport Layer Security (TLS) protocol. 
+
+This authentication process involves:
+
+* **A provisioning certificate*:** an electronic document used to establish the identity of the holder
+* **CIRA:** a protocol used by the remote device to "call home", initiate remote connectivity, with a remote provisioning server.
```

Comment:
CIRA maintains connection to a MPS, not any prov server.  Provisioning communication is all through RPC only when actively running a cmd.

a protocol used by the remote device to "call home", initiate remote connectivity, with a Management Presence Server ~~remote provisioning server~~.